mypy/messages.py

```diff
@@ -575,7 +575,7 @@ def too_few_arguments(self, callee: CallableType, context: Context,
                 msg = 'Missing positional argument'
             else:
                 msg = 'Missing positional arguments'
-            if callee.name and diff:
+            if callee.name and [d for d in diff if d is not None]:
```

Comment:
This could just be `if callee.name and any(diff):` ... Or if that's too cryptic use `any(d is not None for d in diff)`.